application/event_sender/event_sender.py

```diff
@@ -1,68 +1,104 @@
-from typing import Any, Dict
-from aws_lambda_powertools import Logger, Tracer
-from time import time_ns, strftime, gmtime
+from json import dumps
 from os import environ
+from time import gmtime, strftime, time_ns
+from typing import Dict
+
 from aws_embedded_metrics import metric_scope
+from aws_lambda_powertools import Logger, Tracer
 from aws_lambda_powertools.utilities.typing import LambdaContext
+from boto3 import client
 from change_request import ChangeRequest
+from common.dynamodb import put_circuit_is_open
 from common.middlewares import unhandled_exception_logging
-from common.encryption import validate_event_is_signed
-from common.utilities import extract_body
+from common.types import ChangeRequestQueueItem
 
 tracer = Tracer()
 logger = Logger()
 
 
 @tracer.capture_lambda_handler()
 @unhandled_exception_logging
-@validate_event_is_signed
 @logger.inject_lambda_context
 @metric_scope
-def lambda_handler(event: Dict[str, Any], context: LambdaContext, metrics) -> Dict:
+def lambda_handler(event: ChangeRequestQueueItem, context: LambdaContext, metrics) -> Dict:
```

Comment:
This function could do with some refactoring but that can be done in a new ticket later

Reply:
ok will review in separate ticket